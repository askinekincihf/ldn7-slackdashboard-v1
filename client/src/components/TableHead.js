import React from "react";
import slack_logo from "../images/slack_logo.png";
import free_icon_team_new from "../images/free_icon_team_new.png";
import "../pages/Home.css";

const TableHead = () => {
	return (
		<tr className="p-5">
			<th>
				<img className="slack_logo" src={slack_logo} alt="Slack logo" /> Channel
				name
			</th>
			<th>
				<img className="users_icon" src={free_icon_team_new} alt="Users icon" />
				No. of users
			</th>
		</tr>
	);
};

export default TableHead;
