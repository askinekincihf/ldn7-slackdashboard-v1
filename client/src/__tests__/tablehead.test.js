import React from "react";
import renderer from "react-test-renderer";
import TableHead from "../components/TableHead";

describe("TableHead", () => {
	it("should render table head correctly", () => {
		const component = renderer.create(<TableHead />).toJSON();

		expect(component).toMatchSnapshot();
	});
});
