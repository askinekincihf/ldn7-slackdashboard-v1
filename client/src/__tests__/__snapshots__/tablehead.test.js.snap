// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableHead should render table head correctly 1`] = `
<tr
  className="p-5"
>
  <th>
    <img
      alt="Slack logo"
      className="slack_logo"
      src="test-file-stub"
    />
     Channel name
  </th>
  <th>
    <img
      alt="Users icon"
      className="users_icon"
      src="test-file-stub"
    />
    No. of users
  </th>
</tr>
`;
